sutraani/kashika/3/1/31099.txt

```diff
@@ -1 +1 @@
-शक्ल्̥ शक्तौ, षह मर्षणे, अनयोर्धात्वोः यत् प्रत्ययो भवति । शक्यम् । सह्यम् ॥
\ No newline at end of file
+शक्लृ शक्तौ, षह मर्षणे, अनयोर्धात्वोः यत् प्रत्ययो भवति । शक्यम् । सह्यम् ॥
```

Comment:
शकॢ

Reply:
शक्लृ is what I had in my change. Am I missing something?

Reply:
I am not seeing it that way. Copied the शकॢ this time from dhatupata. Hopefully it will be correct